CleverCloud/clever-components
# fix(cc-env-var-editors): set proper values for the editor hidden labels

Fixes #613 

# How to test

- Run locally and go to one of the stories of `cc-env-var-editor-expert`.
- Open your accessibility inspector and go to the `<textarea>` element inside `<cc-input-text>`.
- Check that the "Name" is

>  "Variable editing. Format: VARIABLE_NAME="variable value". Every variable must be separated by a line break."

With Chrome
![image](https://user-images.githubusercontent.com/100240294/200599784-ed352eed-9de0-4568-b797-12cc1e236178.png)

With Firefox (the " are escaped it seems, not sure we can do anything about this)
![image](https://user-images.githubusercontent.com/100240294/200599982-3ce9baab-5330-4ef1-a74c-ab933d53a8a6.png)

- Do the same for `cc-env-var-editor-json`.
